ProcessOptimizer/utils/utils.py

```
@@ -266,13 +272,39 @@ def func(x):
 
     xs = [res.x]
     if n_random_starts > 0:
-        xs.extend(res.space.rvs(n_random_starts, random_state=random_state))
+        if getattr(res.constraints, "sum_equals", None):
```

Should we take other constraint types into account? Possibly just by throwing an error, rather than returning something that doesn't respect the constraints?